scripts/us_census/pep/pep_by_srh/download.sh

```diff
@@ -2,76 +2,89 @@
 
 set -x
 
-rm -rf ./download_files/
+rm -rf ./input_files/
 rm -rf ./process_files/
 rm -rf ./output_files/
 
-mkdir -p ./download_files/1990_2000/county/
-mkdir -p ./download_files/2000_2010/county/
-mkdir -p ./download_files/2010_2020/county/
+mkdir -p ./input_files/1990_2000/county/
+mkdir -p ./input_files/2000_2010/county/
+mkdir -p ./input_files/2010_2020/county/
+mkdir -p ./input_files/2020_2023/county/
 
 mkdir -p ./process_files/agg/1990_2000/county/
 mkdir -p ./process_files/agg/2000_2010/county/
 mkdir -p ./process_files/agg/2010_2020/county/
+mkdir -p ./process_files/agg/2020_2023/county/
 
 mkdir -p ./process_files/as_is/1990_2000/county/
 mkdir -p ./process_files/as_is/2000_2010/county/
 mkdir -p ./process_files/as_is/2010_2020/county/
+mkdir -p ./process_files/as_is/2020_2023/county/
 
-mkdir -p ./download_files/1980_1990/state/
-mkdir -p ./download_files/1990_2000/state/
+mkdir -p ./input_files/1980_1990/state/
+mkdir -p ./input_files/1990_2000/state/
 
 mkdir -p ./process_files/as_is/1980_1990/state/
 mkdir -p ./process_files/as_is/1990_2000/state/
 mkdir -p ./process_files/as_is/2000_2010/state/
 mkdir -p ./process_files/as_is/2010_2020/state/
+mkdir -p ./process_files/as_is/2020_2023/state/
 
 mkdir -p ./process_files/agg/1980_1990/state/
 mkdir -p ./process_files/agg/1990_2000/state/
 mkdir -p ./process_files/agg/2000_2010/state/
 mkdir -p ./process_files/agg/2010_2020/state/
+mkdir -p ./process_files/agg/2020_2023/state/
 
 mkdir -p ./process_files/as_is/1980_1990/national/
 mkdir -p ./process_files/as_is/1990_2000/national/
 mkdir -p ./process_files/as_is/2000_2010/national/
 mkdir -p ./process_files/as_is/2010_2020/national/
+mkdir -p ./process_files/as_is/2020_2023/national/
 
 mkdir -p ./process_files/agg/1980_1990/national/
 mkdir -p ./process_files/agg/1990_2000/national/
 mkdir -p ./process_files/agg/2000_2010/national/
 mkdir -p ./process_files/agg/2010_2020/national/
+mkdir -p ./process_files/agg/2020_2023/national/
 
 mkdir -p ./output_files/
 
 for file in $(cat download_county_files_1990_2000.txt)
 do
     $(curl https://www2.census.gov/programs-surveys/popest/datasets/1990-2000/counties/asrh/${file} \
-          -o ./download_files/1990_2000/county/${file})
+          -o ./input_files/1990_2000/county/${file})
+done
+
+for file in $(cat download_county_files_2020_2023.txt)
+do
+    $(curl https://www2.census.gov/programs-surveys/popest/datasets/2020-2023/counties/asrh/${file} \
```

Comment:
no need to wrap command with `$(...)`
Also safer to enclose url and arguments in double quotes `""`.

can do
```
curl "https://www2.census.gov/programs-surveys/popest/datasets/2020-2023/counties/asrh/${file}" \
  "/input_files/2020_2023/county/${file}"
```

Reply:
Thanks for checking Ajai. We've moved away from using download.sh